heartbeat/LVM-activate

```diff
@@ -0,0 +1,830 @@
+#!/bin/bash
```

Comment:
Looks like no bash-specific features are used. Better go with POSIX sh (/bin/sh).

Reply:
Dejan,
Will have a try!

thanks very much for your review.

Reply:
Hi Dejan,

After changing to /bin/sh, the checker gives some warnings:

```
In LVM-activate line 694:
	[[ -z ${LV} ]] && vol=${VG} || vol=${VG}/${LV}
        ^-- SC2039: In POSIX sh, [[ ]] is undefined.
...
In LVM-activate line 728:
	local vol
        ^-- SC2039: In POSIX sh, 'local' is undefined.
```

Sorry for my laziness, I'd still use /bin/bash, though I know it's easy to change. But, the previous new RA lvmlockd already uses /bin/bash. Someone can easily change to /bin/sh together if there's a real problem for some cases.